### PR TITLE
Use consistent UI for apikeys form

### PR DESCRIPTION
**Pull request by comzeradd**
Added fieldsets around form inputs and button to have consistent UI with other forms (eg. profile edit).

### CODE REVIEW

**Comment by johngian** [comment]
Please reference this [1] bug without adding the word fix, just add ``[Bug <>]`` prefix.

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1011240

File: mozillians/templates/phonebook/apikeys.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <header>
   <h1>{{ _('Mozillians API') }}</h1>
 </header>
-<h2>{{ _('API Version 1') }}</h4>
+<h2>{{ _('API Version 1') }}</h2>
 <p class="field_description">
   {% trans api_schema_url='http://mozillians.readthedocs.org/en/latest/api.html',
```
Comment (johngian) [inline mozillians/templates/phonebook/apikeys.html]: Could you also fix this broken URL to: http://mozillians.readthedocs.org/en/latest/api/api.html

File: mozillians/templates/phonebook/apikeys.html [hunk 2]
```diff
@@ -85,14 +85,28 @@
     {% endfor %}
   </ul>
 
-  <strong>Register a new app</strong>
+  <p>
```
Comment (johngian) [inline mozillians/templates/phonebook/apikeys.html]: For consistency, it makes sense to use the same format for ``Your apps`` string.

**Comment by johngian** [comment]
r+wc

**Comment by comzeradd** [comment]
Commit amended

**Pull request merged**